### PR TITLE
update rpc contract with latest changes

### PR DESCRIPTION
**Pull request by LukasJenicek**

### CODE REVIEW

**Review by corbanbrook — APPROVED** [review]

File: packages/api/src/api.gen.ts [hunk 1]
```diff
@@ -396,6 +422,7 @@ export interface SendPasswordlessLinkReturn {
   status: boolean
 }
 export interface FriendListArgs {
+  nickname: string
```
Comment (corbanbrook) [inline packages/api/src/api.gen.ts:425]: nickname should not be a required property

Comment (LukasJenicek) [reply packages/api/src/api.gen.ts:425]: Passing an empty string will work.

Comment (corbanbrook) [reply packages/api/src/api.gen.ts:425]: Thats what im doing for now, but we should correct this

Comment (pkieltyka) [reply packages/api/src/api.gen.ts:425]: yes, change the RIDL to make it optional @LukasJenicek

Comment (LukasJenicek) [reply packages/api/src/api.gen.ts]: Totally forgot about this one. It's merged  `stack/api` :https://github.com/0xsequence/stack/pull/774 and deployed: https://buildkite.com/horizon/sequence-stack-build/builds/1068

Comment (LukasJenicek) [reply packages/api/src/api.gen.ts]: I just did not like the idea that we have to reference string value by pointer, but I think there is nothing I can do about it

**Pull request closed without merging**